app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,4 @@
+@import 'bootstrap/scss/bootstrap';
+@import url('https://fonts.googleapis.com/css?family=Lora:400,700,400italic,700italic');
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800');
+@import 'clean-blog';
```

Comment:
Why do you need to import `clean-blog` here?

Reply:
I removed the `clean-blog` file and use the corresponding styles for each view.